agent/api/task/task.go

```diff
@@ -3687,3 +3687,33 @@ func (task *Task) HasActiveContainers() bool {
 	}
 	return false
 }
+
+// IsManagedDaemonTask will check if a task is a non-stopped managed daemon task
+// TODO: We'll probably want to also clean up all of the STOPPED managed daemon task from
```

Comment:
nit -- the todo sentence seems to be missing the last part...
"from..." 
from what?

Reply:
no longer needed, same todo is in `data.go`